File: problemas/ad-hoc/2381/2381.js
let input = require("fs").readFileSync("/dev/stdin", "utf8");
let lines = input.trim().split("\n");

let [N, K] = lines.shift().trim().split(' ').map(x => parseInt(x));

let alunos = [];
for (let i = 0; i < N; ++i) {
    let aluno = lines.shift().trim();
    alunos.push(aluno);
}

alunos.sort();
console.log(alunos[K - 1]);

File: problemas/ad-hoc/2381/2381.rs
use std::io;

fn main() {
    let mut input = String::new();
    io::stdin().read_line(&mut input);
    let parts: Vec<&str> = input.split(" ").collect();
    let n: i32 = parts[0].trim().parse().unwrap();
    let k: i32 = parts[1].trim().parse().unwrap();

    let mut alunos: Vec<String> = Vec::new();
    for _ in 0..n {
        let mut input2 = String::new();
        io::stdin().read_line(&mut input2);
        let linha: String = input2.trim().parse().unwrap();
        alunos.push(linha);
    }
    alunos.sort();

    println!("{}", alunos[(k - 1) as usize]);
}
